src/ToDoItem.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import User from './User';
+
+const ToDoItem = ({ todo: { title, user, completed } }) => (
+  <tr style={{ borderBottom: '1px solid grey' }}>
+    <td style={{ backgroundColor: completed ? '#ADFF2F' : '#FFA07A' }}>
+      {title}
+    </td>
+    <User user={user} />
```

Comment:
три подряд слова user выглядит не очень читаемо
Называй переменные по-разному. Пример:
```suggestion
    <User userData={user} />
```